sycl/test-e2e/bindless_images/device_to_device_copy.cpp

```diff
@@ -0,0 +1,99 @@
+// REQUIRES: linux
+// REQUIRES: cuda
+
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+#include <iostream>
+#include <sycl/sycl.hpp>
+
+// Uncomment to print additional test information
+#define VERBOSE_PRINT
```

Comment:
```suggestion
// #define VERBOSE_PRINT
```

Reply:
Verbosity for CI failure? Is this ready for reviews?

Reply:
>is this ready for reviews?

it's not the new CI failure invalidated the previous fix.